core/src/iosMain/kotlin/com/revenuecat/purchases/kmp/Purchases.ios.kt

```diff
@@ -1,288 +1,370 @@
-@file:Suppress("EXTENSION_SHADOWED_BY_MEMBER")
+@file:Suppress("ACTUAL_ANNOTATIONS_NOT_MATCH_EXPECT")
```

Comment:
Hmm curious which members aren't matching the expect? Just wondering since this seems like a useful warning to have. If it is needed, might be good adding a comment explaining the need.

Reply:
I removed the warning and with that the need for this suppression!
 
It suppressed a warning about a missing `@JvmStatic` annotation (which was present in the commonMain source set):
>Annotation `@JvmStatic` is missing on actual declaration.  
  
However that annotation cannot be set in the iosMain source set, so it was impossible to satisfy the linter, basically. 

Removing the annotations in the commonMain source set and only keeping them in androidMain works just as well, but without warnings. The only thing I could not get to work is `@JvmOverloads` on `canMakePayments()`. We can try that again later imo.